client/elements/menus/sc-language-menu.html

```diff
@@ -218,13 +218,13 @@
           }
 
           // If author name is too long it looks really bad in paper-item so just cut it.
-          _shortenAuthor(author) {
-              if (author.length > 16) {
-                  return `${author.substring(0, 15)}...`;
-              } else {
-                  return author;
-              }
-          }
+          // _shortenAuthor(author) {
+          //     if (author.length > 16) {
+          //         return `${author.substring(0, 15)}...`;
+          //     } else {
+          //         return author;
+          //     }
+          // }
```

Comment:
Are we gonna need this in the future? [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) says no :)

Reply:
Oops ... forgot to delete those bits!